habitat/tasks/rearrange/sub_tasks/pick_task.py

```diff
@@ -277,4 +277,7 @@ def reset(self, episode: Episode):
 
         self._targ_idx = sel_idx
 
-        return self._get_observations(episode)
+        if fetch_observations:
+            return self._get_observations(episode)
+        else:
```

Comment:
No need for else after return?

Reply:
I believe flake needs an explicit return at the end of the function.

Reply:
@ASzot Yeah, but the else is unnecessary.